### ISSUE TITLE
CLI emits warnings on valid Babel options

### ISSUE THREAD

**Issue by herrstucki** [lingui/js-lingui#566]
**Describe the bug**
When I use [documented Babel options](https://babeljs.io/docs/en/options#config-loading-options), I get a warning. *The configuration still works though*, it just seems that your validation is too strict.

**To Reproduce**

Example `.linguirc` file:

```
{
  "extractBabelOptions": {
    "configFile": "./app/babel.config.js"
  }
}
```

Produces this warning (repeatedly, probably for each source file?):

```
● Validation Warning:

  Unknown option "extractBabelOptions.configFile" with value "./app/babel.config.js" was found.
  This is probably a typing mistake. Fixing it will remove this message.

See https://lingui.js.org/ref/conf.html for a list of valid options
```

**Expected behavior**

No warnings for valid Babel options – maybe not validate Babel options at all and just hand them off to Babel?

**Additional context**

- jsLingui version `2.8.3`
- Babel version `@babel/core@7.4.5`

**Comment by herrstucki** [comment]
Still an issue in v2

**Comment by tricoder42** [comment]
Hey @herrstucki, there was just new major version released, 3.0.0. Feel free to send a PR, but please also backport it to branch `stable-2.x` if you want to release the patch in v2.

**Comment by herrstucki** [comment]
@tricoder42 Awesome! I'll do that when I find a minute but I'm not sure when that will be ;) So feel free to close the issue anyway, since it's not blocking anyone.
